spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -84,4 +84,23 @@
       expect(verification_array.empty?).to eq true
     end
   end
+
+  context '#identify_druid' do
+    let(:same_druid) { 'bz514sm9647' }
+    let(:same_druid_path) { 'spec/fixtures/storage_root01/moab_storage_trunk/bz/514/sm/9647/bz514sm9647' }
+    let(:storage_obj1) { Moab::StorageObject.new(same_druid, same_druid_path) }
+    let(:storage_obj_validator1) { described_class.new(storage_obj1) }
+    let(:different_druid) { 'jj925bx9565' }
+    let(:different_druid_path) { 'spec/fixtures/storage_root01/moab_storage_trunk/jj/925/bx/9565/jj925bx9565' }
+    let(:storage_obj2) { Moab::StorageObject.new(different_druid, different_druid_path) }
+    let(:storage_obj_validator2) { described_class.new(storage_obj2) }
+
+    it 'returns empty errors array when druid is the same' do
+      expect(storage_obj_validator1.identify_druid).to eq []
+    end
+
+    it 'returns the right error code when druid is not the same' do
+      expect(storage_obj_validator2.identify_druid).to eq [{ 9=>'manifestInventory object_id does not match druid' }]
```

Comment:
feels to me like it'd be slightly clearer to use `Moab::StorageObjectValidator::DRUID_MISMATCH` instead of the underlying int code.